shell/common/skia_event_tracer_impl.cc

```diff
@@ -183,22 +186,67 @@ class FlutterEventTracer : public SkEventTracer {
     trace_release_context(trace_context);
 
 #else   // defined(OS_FUCHSIA)
+    std::vector<const char*> arg_names;
+    std::vector<std::string> arg_values;
```

Comment:
It's probably a bit out of scope of this CL, but taking a step back, why do all arg values have to be converted to strings here?  Why can't (e.g.) ints stay ints, instead of being converted to strings?  Before this CL lands, I think it's worth examining what it would take to not have to convert all argument values to strings.  Also note that the Fuchsia version preserves the types of the Skia trace arguments.